Pod/Classes/ios/NYTPhotosOverlayView.m

```diff
@@ -49,6 +50,10 @@ - (void)layoutSubviews {
     }];
     
     [super layoutSubviews];
+
+    if ([self.captionView respondsToSelector:@selector(setPreferredMaxLayoutWidth:)]) {
```

Comment:
This is all implementation detail! And I think I am sufficiently convinced not to micro-optimize this for now (though I suspect that [this cache lookup](https://github.com/opensource-apple/objc4/blob/17d47b3989923007fa052f346b05a32fca389465/runtime/objc-runtime-new.mm#L4996) will almost always succeed); see https://github.com/NYTimes/NYTPhotoViewer/commit/162e0ba0622b61f394532bfd822205ceec3b0f42